exercises/static/exercises/drone_cat_mouse/web-template/exercise.py

```diff
@@ -146,21 +146,21 @@ def generate_modules(self):
         # motors# hal_module.HAL.motors = imp.new_module("motors")
 
         # Add HAL functions
-        hal_module.HAL.get_frontal_image = self.hal.get_frontal_image
```

Comment:
get back HAL when new drone_wrapper backend is ready

Reply:
sure will make the changes as soon as the drone wrapper is ready and functioning for rotors